tests/generator/generator.cpp

```diff
@@ -24,437 +24,58 @@ static const std::string EMPTY_STRING;
 
 TEST(Generator, emptyModel)
 {
-    libcellml::ModelPtr model = libcellml::Model::create();
-    libcellml::GeneratorPtr generator = libcellml::Generator::create();
+    libcellml::ModelPtr model = libcellml::Model::create("empty_model");
+    libcellml::AnalyserPtr analyser = libcellml::Analyser::create();
 
-    generator->processModel(model);
+    analyser->analyseModel(model);
 
```

Comment:
is this the equivalent of setting `nullptr` as the argument, or is that tested somewhere too?

Reply:
No, it's not the same. `analyser->analyseModel(nullptr);` is tested in `TEST(Analyser, coverage)`.